app/assets/javascripts/application.js

```diff
@@ -17,4 +17,7 @@
 //= require jquery-ui/datepicker
 //= require jquery-ui/accordion
 //= require jquery-ui-timepicker-addon
+//= require datetimepicker
+//= require jquery-ui
+//= require bootstrap
```

Comment:
According to [Bootstrap](https://github.com/twbs/bootstrap-sass#a-ruby-on-rails) (toward the bottom of that section):
> bootstrap-sprockets and bootstrap should not both be included in application.js.
>
> bootstrap-sprockets provides individual Bootstrap Javascript files (alert.js or dropdown.js, for example), while bootstrap provides a concatenated file containing all Bootstrap Javascripts.

They [link](https://github.com/twbs/bootstrap-sass/issues/829#issuecomment-75153827) to an issue thread explaining it.